### PR TITLE
dbus-listener: add requesting all values for init

### PR DESCRIPTION
**Pull request by sbender9**

### CODE REVIEW

File: venusToDeltas.js [hunk 1]
```diff
@@ -269,5 +276,5 @@ const fluidTypeMapping = {
 
 function getFluidType(typeId) {
   debug(`getFluidType ${typeId} ${fluidTypeMapping[typeId]}`)
-  return fluidTypeMapping[typeId] | 'unknown';
+  return fluidTypeMapping[typeId] || 'unknown';
```
Comment (mpvader) [inline venusToDeltas.js]: hi @sbender9 , this is a bug-fix thats not related to the get root at init? or is it? What does it fix?

The rest makes sense & looks good.

**Comment by mpvader** [comment]
ps one other trick (just in case you didn't know it already): if you add this line in the commit message:

`closes https://github.com/mpvader/venus-signalk/issues/8`

github will automatically close that issue when I put the commit in master.

**Comment by mpvader** [comment]
hm I see something else now, in your commit you've left the function `getDeviceInstanceForService` in place, but it isn't used anymore. At least it shouldn't be: the one place where its still used needs to be replaced by init_service() as well. Or did you not do that on purpose?

Compare your commit with this one:
https://github.com/mpvader/venus-signalk/commit/edc534c5f5df0fcd815ed6053a319c0c157b40d8#diff-b2af702f54713360eb7a6772e85e248aL46

Otherwise, why not just let my commit as is, and only do another extra one on top that adds the rest of the changes?

I can squash them if necessary, but at least it makes the review easier.

File: venusToDeltas.js (hunk 1 above)
Comment (sbender9) [reply venusToDeltas.js]: Related only in that I needed the fix to be able to test this since tanks are the only thing get root works with

**Pull request closed without merging**